internal/kustomize/kustomize.go

```diff
@@ -26,6 +26,7 @@ func Render(
 	path string,
 	images []string,
 	enableHelm bool,
+	loadRestrictor string,
```

Comment:
Because I can foresee the number of options continuing to grow, I think we should receive `cfg Config` as the last argument here instead of receiving `enableHelm` and `loadRestrictor` separately. Same for `BuildKustomizeOptions()`.